### PR TITLE
Fix #1399 : Added 'ignore conda-verify error message' in FAQ

### LINKED ISSUE #1399: Document ignoring conda-verify
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

During the build process, the following error is shown: `Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.`

This can safely be ignored and it would be good to have this documented somewhere. 

Prompted by: https://gitter.im/conda-forge/conda-forge.github.io?at=6079b460dc24646812c5cf4b

### PR DESCRIPTION
**Pull request by ForgottenProgramme**
PR Checklist:

- [X] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [X] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [X] put any other relevant information below

Fixes #1399

### CODE REVIEW

**Comment by ForgottenProgramme** [comment]
@BastianZim 
kindly have a look at this please?

**Comment by BastianZim** [comment]
@ForgottenProgramme Thanks for the PR and sorry for not reviewing it earlier. I just added two small points above.

File: src/maintainer/maintainer_faq.rst [hunk 1]
```diff
@@ -86,5 +86,24 @@ FAQ
...
     For more details visit `this <https://boa-build.readthedocs.io/en/latest/mambabuild.html>`_ page. 
+    
+    .. _mfaq_conda_verify:
+
+:ref:`(Q) <mfaq_conda_verify>` **I am seeing** ``Importing conda-verify failed.`` **error message during build. What do I do?**
+
+   ``Importing conda-verify failed. Please be sure to test your packages. conda install conda-verify to make this message go away.``
+You are seeing this error message because by default, conda-build uses conda-verify to ensure that your recipe and package meet some minimum sanity checks. 
+This message can be safely ignored. 
+Additionally, if you wish to make this message go away, you may disable conda-verify.
```
[the remark this answers is not in the document]
Comment (ForgottenProgramme) [reply src/maintainer/maintainer_faq.rst:97]: Makes sense!

File: src/maintainer/maintainer_faq.rst [hunk 2]
```diff
@@ -86,5 +86,24 @@ FAQ
...
       - ``conda mambabuild myrecipe``
     For more details visit `this <https://boa-build.readthedocs.io/en/latest/mambabuild.html>`_ page. 
+    
+    .. _mfaq_conda_verify:
+
+:ref:`(Q) <mfaq_conda_verify>` **I am seeing** ``Importing conda-verify failed.`` **error message during build. What do I do?**
+
+   ``Importing conda-verify failed. Please be sure to test your packages. conda install conda-verify to make this message go away.``
+You are seeing this error message because by default, conda-build uses conda-verify to ensure that your recipe and package meet some minimum sanity checks. 
+This message can be safely ignored.
```
[the remark this answers is not in the document]
Comment (ForgottenProgramme) [reply src/maintainer/maintainer_faq.rst:96]: Thankyou Bastian. :)
I have one doubt though.
I don't understand what we mean by 'conda-forge doesn't use it'. 
So, can I write, "This message can be safely ignored because conda-forge doesn't use conda-verify" ?

[the remark this answers is not in the document]
Comment (BastianZim) [reply src/maintainer/maintainer_faq.rst:96]: Yeah, something along those lines.

Conda-verify is an optional dependency of conda-build so whoever installs conda-build can either install conda-verify or not (Which is the case in the conda-forge infrastructure): https://github.com/conda/conda-build/blob/3b99b2222a067e113a2282926871cd1e5406ee2b/conda.recipe/meta.yaml#L30

During execution, conda-build will try to import conda-verify and run it, otherwise, if conda-build cannot find (Meaning import) conda-verify, it will just skip the verification.
https://github.com/conda/conda-build/blob/3b99b2222a067e113a2282926871cd1e5406ee2b/conda_build/build.py#L1683-L1688

Since the scripts used to set up the conda-forge CI infrastructure and local test environment do not install conda-verify, the latter is the case.

So long story short: The conda-forge infrastructure does not install conda-verify as it is not needed and therefore that part is skipped. :)

[the remark this answers is not in the document]
Comment (ForgottenProgramme) [reply src/maintainer/maintainer_faq.rst:96]: Wowwwww!
Gottt ittt! <3

[the remark this answers is not in the document]
Comment (ForgottenProgramme) [reply src/maintainer/maintainer_faq.rst:96]: Thanks so explaining so well.

File: src/maintainer/maintainer_faq.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (viniciusdc) [reply src/maintainer/maintainer_faq.rst:97]: Yup, normally we just ignore the message

File: src/maintainer/maintainer_faq.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (viniciusdc) [reply src/maintainer/maintainer_faq.rst:96]: Thanks for the comment! I couldn't have done it better

[the remark this answers is not in the document]
Comment (BastianZim) [reply src/maintainer/maintainer_faq.rst:96]: Happy to help :)

**Comment by ForgottenProgramme** [comment]
Updated as suggested.
